app/models/maintenance_stats/stats/bitbucket/issue_stats.rb

```diff
@@ -0,0 +1,81 @@
+module MaintenanceStats
+    module Stats
+        module Bitbucket
+            class IssueRates < MaintenanceStats::Stats::BaseStat
+                def count_issues(issues, state: nil)
+                    if state.nil?
+                      issues.size
+                    else
+                      issues.where(state: state).size
+                    end
+                  end
+            
+                  def closed_issues_count
+                    return @closed_issues_count unless @closed_issues_count.nil?
+            
+                    @closed_issues_count = count_issues(issues, state: "closed")
+                    @closed_issues_count
```

Comment:
No need for this line - the value of the assignment is the assigned value

The usual pattern would be something like:
```ruby
@closed_issues_count ||= count_issues(issues, state: "closed")